### PR TITLE
fix(nuxt): Add e2e-ci and serve-static targets

### PR DESCRIPTION
**Pull request by ndcunningham**
Add missing targets for nuxt.

### CODE REVIEW

File: docs/generated/packages/nuxt/documents/overview.md [hunk 1]
```diff
@@ -78,3 +82,21 @@ nx g @nx/nuxt:app my-app
 Once you are ready to deploy your Nuxt application, you have absolute freedom to choose any hosting provider that fits your needs.
 
 We have detailed [how to deploy your Nuxt application to Vercel in a separate guide](/recipes/nuxt/deploy-nuxt-to-vercel).
+
+### E2E testing
+
+You can perform end-to-end (E2E) testing on static HTML files using a test runner like Cypress. When you create a Next.js application, Nx automatically creates a `serve-static` target. This target is designed to serve the static HTML files produced by the build command.
```
Comment (isaacplmann) [inline docs/generated/packages/nuxt/documents/overview.md]:
```suggestion
You can perform end-to-end (E2E) testing on static HTML files using a test runner like Cypress. When you create a Nuxt application, Nx automatically creates a `serve-static` target. This target is designed to serve the static HTML files produced by the build command.
```

File: docs/shared/packages/nuxt/nuxt-plugin.md [hunk 2]
```diff
@@ -78,3 +82,21 @@ nx g @nx/nuxt:app my-app
 Once you are ready to deploy your Nuxt application, you have absolute freedom to choose any hosting provider that fits your needs.
 
 We have detailed [how to deploy your Nuxt application to Vercel in a separate guide](/recipes/nuxt/deploy-nuxt-to-vercel).
+
+### E2E testing
+
+You can perform end-to-end (E2E) testing on static HTML files using a test runner like Cypress. When you create a Next.js application, Nx automatically creates a `serve-static` target. This target is designed to serve the static HTML files produced by the build command.
```
Comment (isaacplmann) [inline docs/shared/packages/nuxt/nuxt-plugin.md]:
```suggestion
You can perform end-to-end (E2E) testing on static HTML files using a test runner like Cypress. When you create a Nuxt application, Nx automatically creates a `serve-static` target. This target is designed to serve the static HTML files produced by the build command.
```

**Review by isaacplmann — CHANGES_REQUESTED** [review]
How is `build-static` different from `build`?  I thought `build` was already static.

File: docs/generated/packages/nuxt/documents/overview.md [hunk 3]
```diff
@@ -78,3 +82,21 @@ nx g @nx/nuxt:app my-app
...
+### E2E testing
+
+You can perform end-to-end (E2E) testing on static HTML files using a test runner like Cypress. When you create a Nuxt application, Nx automatically creates a `serve-static` target. This target is designed to serve the static HTML files produced by the build command.
+
+This feature is particularly useful for testing in continuous integration (CI) pipelines, where resources may be constrained. Unlike the `dev` and `start` targets, `serve-static` does not require a Nuxt's Nitro server to operate, making it more efficient and faster by eliminating background processes, such as file change monitoring.
+
+To utilize the `serve-static` target for testing, run the following command:
+
+```shell
+nx serve-static my-next-app-e2e
```
Comment (isaacplmann) [inline docs/generated/packages/nuxt/documents/overview.md:95]:
```suggestion
nx serve-static my-nuxt-app-e2e
```

File: docs/shared/packages/nuxt/nuxt-plugin.md [hunk 4]
```diff
@@ -78,3 +82,21 @@ nx g @nx/nuxt:app my-app
...
+### E2E testing
+
+You can perform end-to-end (E2E) testing on static HTML files using a test runner like Cypress. When you create a Nuxt application, Nx automatically creates a `serve-static` target. This target is designed to serve the static HTML files produced by the build command.
+
+This feature is particularly useful for testing in continuous integration (CI) pipelines, where resources may be constrained. Unlike the `dev` and `start` targets, `serve-static` does not require a Nuxt's Nitro server to operate, making it more efficient and faster by eliminating background processes, such as file change monitoring.
+
+To utilize the `serve-static` target for testing, run the following command:
+
+```shell
+nx serve-static my-next-app-e2e
```
Comment (isaacplmann) [inline docs/shared/packages/nuxt/nuxt-plugin.md:95]:
```suggestion
nx serve-static my-nuxt-app-e2e
```

**Comment by ndcunningham** [comment]
> How is `build-static` different from `build`? I thought `build` was already static.

It requires the nuxt serve to run. i.e. there are no static files running `build-static` runs `nuxt build --prerender` that creates statics files inside the `{projectRoot}/dist`. I can add this to the documentation as well to make it more clear

**Review by jaysoo — APPROVED** [review]

**Comment by isaacplmann** [comment]
Yeah, if you explain what `build-static` is in the docs, that would be helpful.

**Review by isaacplmann — APPROVED** [review]

**Pull request merged**